cmd/gokey/main.go

```diff
@@ -110,6 +110,8 @@ func logFatal(format string, args ...interface{}) {
 func main() {
 	flag.Parse()
 
+	pass = os.Getenv("GOKEY_MASTER")
```

Comment:
if I supply a password via cmdline, but don't set `GOKEY_MASTER` won't this reset my pass?